.github/workflows/ci.yml

```diff
@@ -21,6 +21,7 @@ env:
   SERVER_PROJECT_ID: 'github-ci-server'
   CLIENT_PROJECT_ID: 'github-ci-app-0'
   BIGQUERY_DATASET_ID: 'audit_logs.audit_abcxyz_data_access'
+  AUDIT_CLIENT_TEST_PRIVATE_KEY_PATH: './integration/testrunner/private_key.json'
```

Comment:
I don't think we need to add "AUDIT_CLIENT_TEST" prefix here

Reply:
I removed this one, and currently only use the one for integration test only.